raiden/utils/mediation_fees.py

```diff
@@ -16,7 +16,9 @@ def ppm_fee_per_channel(per_hop_fee: ProportionalFeeAmount) -> ProportionalFeeAm
     for how to get to this formula.
     """
     per_hop_ratio = Fraction(per_hop_fee, 10 ** 6)
-    return ProportionalFeeAmount(round(per_hop_ratio / (per_hop_ratio + 2) * 10 ** 6))
+    return ProportionalFeeAmount(
+        round(Fraction(per_hop_ratio / (per_hop_ratio + 2) * 10 ** 6))  # type: ignore
```

Comment:
`round(Fraction(value))` brings no improvement over `round(value)`. Why the change here?

If you want to use Fraction to increase the accuracy, you need to use `Fraction(num, den)`. But when rounding anyway, I don't see the benefit for that.

Reply:
Sorry, this was a leftover being there despite the type-ignore